<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>requestForm</class>
 <widget class="QWidget" name="requestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Свойства запроса</string>
   </property>
   <widget class="QComboBox" name="attributeComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Знак зодиака</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Признак запроса   --&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск по фамилии</string>
    </property>
    <widget class="QLineEdit" name="lastNameLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия              --&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск по знаку зодиака</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Знак зодиака   --&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="zodiacSignComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Овен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Телец</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Близнецы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Рак</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лев</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дева</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Весы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Скорпион</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Стрелец</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Козерог</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Водолей</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Рыбы </string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
